source/languages/en/riak/community/index.md

```diff
@@ -11,116 +11,79 @@ moved: {
 
-Basho also maintains a frequently updated page of [[community-developed
-projects|Community Projects]], including GUIs, backup tools, and more.
+### Code
 
-### Riak on IRC
+Basho uses GitHub for the development of all of our
+code. Our core products are developed [under the Basho organization](http://github.com/basho/). We also have [Basho Labs](http://github.com/basho-labs/), where we curate the rich ecosystem of contributors from our community. We also make use of GitHub for the submission and integration of all internal code and community pull requests. If you have general feedback beyond the scope of a single repo, open an issue under [The Riak Community](https://github.com/basho-labs/the-riak-community/issues).
 
-The **#riak** IRC room on irc.freenode.net is a great place for realtime
-help with your Riak issues and questions.
+### Events on Meetup.com
 
-### The Basho Blog
+We have local meetups everywhere from Paris to San Francisco and back again. To find the one closest to you, search for [Riak on Meetup.com](http://riak.meetup.com/).
 
-Members of the Basho Team blog frequently on [[The Basho
-Blog|http://blog.basho.com]].
+### Riak on IRC
 
-### The Riak Recap
+The [#riak IRC room on irc.freenode.net](http://irc.lc/freenode/riak) is a great place for realtime help with your Riak issues and questions.
 
-The Riak Recap is a periodic email to the Riak Users Mailing List to
-keep list subscribers up-to-date on what’s happening in and around the
-Riak Community.
+### The Basho Blog
 
-### Riak Function Contrib
+Members of the Basho Team blog frequently on [The Basho
+Blog](http://blog.basho.com).
 
-Riak Function Contrib is a home for community contributed MapReduce,
-Pre- and Post-Commit Hook, and utility functions.
+### Videos
 
-* [contrib.basho.com](http://contrib.basho.com)
-* [Riak Function Contrib Repo on
-  GitHub](https://github.com/basho/riak_function_contrib)
-* [Introducing Riak Function Contrib (from the Basho
-  Blog)](http://blog.basho.com/2010/12/02/introducing-riak-function-contrib/)
+You can watch all the Basho curated videos [on YouTube](https://www.youtube.com/user/BashoTechnologies) and a few others on the [Basho Vimeo Page](http://vimeo.com/bashotech/videos).
 
-### Riak on Facebook
 
-We have a [Facebook
-Page](http://www.facebook.com/pages/Riak/143599935667217) where you can
-get involved.
+### Other Social 
 
-### Riak on LinkedIn
+We are easy to find on social media sites. Like us on [Facebook](http://www.facebook.com/pages/Riak/143599935667217), chat on the [Riak
+LinkedIn Group](http://www.linkedin.com/groups?mostPopular=&gid=2913983), [follow us on Twitter](http://twitter.com/basho). We have a growing following [on Reddit as well](reddit.com/r/Riak/).
 
-There is also a [Riak Group on
-LinkedIn](http://www.linkedin.com/groups?mostPopular=&gid=2913983).
+## Programs 
+Our platforms are complemented by the amazing community efforts of Basho and the greater contributors. Here are a few you don't want to miss.
```

Comment:
"Our platforms are complemented by the amazing efforts of the Basho team and the greater contributor community."